docs/how-tos/validate-nexus-file.md

```diff
@@ -0,0 +1,802 @@
+
+```
+findings
+======= ====== ========== ======================================
+address status test       comments                              
+======= ====== ========== ======================================
+/entry  ERROR  known NXDL NXopt: unrecognized NXDL specification
+======= ====== ========== ======================================
+
+
+summary statistics
+======== ===== =========================================================== =========
+status   count description                                                 (value)  
+======== ===== =========================================================== =========
+OK       148   meets NeXus specification                                   100      
+NOTE     0     does not meet NeXus specification, but acceptable           75       
+WARN     0     does not meet NeXus specification, not generally acceptable 25       
+ERROR    1     violates NeXus specification                                -10000000
+TODO     16    validation not implemented yet                              0        
+UNUSED   0     optional NeXus item not used in data file                   0        
+COMMENT  0     comment from the punx source code                           0        
+OPTIONAL 213   allowed by NeXus specification, not identified              99       
+         --                                                                         
+TOTAL    378                                                                        
+======== ===== =========================================================== =========
+```
+
+The last error message:
+
+```
+======= ====== ========== ======================================
+/entry  ERROR  known NXDL NXopt: unrecognized NXDL specification
+======= ====== ========== ======================================
+```
+
+can be ignored and is a bug right now. If this is the only Error message, then your NeXus file is compliant with the NeXus definitions and you can share and publish your data.
+
+# 3.4. Further Notes
+
+1. Punx only uses the NeXus definiton from the NIAC [NeXus definiton from the NIAC](<https://manual.nexusformat.org/>). The use of the [FAIRmat NeXus definition](<https://fairmat-nfdi.github.io/nexus_definitions/index.html#>) is not possible right now.
+
+2. [Other punx commands are availble](<https://punx.readthedocs.io/en/latest/overview.html#>)
+
+3. [More details for installation](<https://punx.readthedocs.io/en/latest/install.html>)
+
+4. [Github project](<https://github.com/prjemian/punx>)
+
+
+
+
+# Summary
+
+This tutorial showed:
+
+1. 3 different tools for NeXus file validation
+
+2. How to install these tools
+
+3. How to use them via Examples
+
+### Recommended methods:
+
+As pynxtools verify_nexus method is right now in development, [not all situations are covered right now](/testing-validation-tools.md). Therefore, the most reliable method right now is a combination of _Human Manual Validation_ + _Software solutions_.
+
+### Pynxtools Parsers:
```

Comment:
Either remove this part or, if you want to keep it, simply make a link to ../reference/plugins.md (in order to keep single source principle).

Reply:
yea, put in a reference and just kept the line 767.

Reply:
rechecked, is included